### ISSUE TITLE
Resume pretraining. 

### ISSUE THREAD

**Issue by opencomvis** [lightly-ai/lightly#1347]
Hi, thanks for this nice work. 
I am using one of your tutorial codes. After the training has finished, i realized that it might be beneficial to just keep the model training for 200 epochs more. So, I wanted to continue the training starting from the last checkpoint saved! 
Do you have scripts that explain how to resume training? 
and will that affect the training and potentially harm the already learnt embeddings ?

**Comment by philippmwirth** [comment]
Hi @opencomvis , thanks for using Lightly!

You can load the checkpoint using torch's `load_state_dict` as outlined [here](https://pytorch.org/tutorials/beginner/basics/saveloadrun_tutorial.html?highlight=checkpoint%20load). Please note that our checkpoints to not include the state of the optimizer etc so you're going to want to pay attention to that. If you're following one of our pytorch-lightning tutorials you can pass the path to the checkpoint to the `fit` function (see [here](https://lightning.ai/docs/pytorch/latest/common/trainer.html#fit)).

In our experience, embeddings become better the longer training goes on even if the loss starts to converge. Just don't mix embeddings of the model at two different checkpoints.
